content/en/news/releases/1.11.x/announcing-1.11/upgrade-notes/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.11.0.
+weight: 20
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+When you upgrade from Istio x.y.(z-1) to Istio 1.11.0, you need to consider the changes on this page.
```

Comment:
```suggestion
When you upgrade from Istio 1.10.0 to Istio 1.11.0, you need to consider the changes on this page.
```

Reply:
For future reference, this should be substituted by the `--oldRelease` on `gen-release-notes`